api/krusty/remoteloader_test.go

```diff
@@ -188,14 +188,6 @@ resources:
 			kustomization: `
 resources:
 - file://$ROOT/simple.git?timeout=500
-`,
-			expected: simpleBuild,
-		},
-		{
-			name: "triple slash absolute path",
-			kustomization: `
-resources:
-- file:///$ROOT/simple.git
```

Comment:
It turns out `$ROOT` already contains a slash, so this test case used four slashes not three, and that's not valid.